CC/Sounder/utils.cc

```diff
@@ -108,13 +108,14 @@ std::vector<std::vector<size_t>> Utils::loadSymbols(
     std::vector<std::string> frames, char sym)
 {
     std::vector<std::vector<size_t>> symId;
-    int frameSize = frames.size();
+    size_t frameSize = frames.size();
     symId.resize(frameSize);
-    for (int f = 0; f < frameSize; f++) {
+    for (size_t f = 0; f < frameSize; f++) {
         std::string fr = frames[f];
         for (size_t g = 0; g < fr.size(); g++) {
             if (fr[g] == sym) {
                 symId[f].push_back(g);
+                printf("%c: pushing %zu back to vector %zu\n", sym, g, f);
```

Comment:
You could move this to the logging interface.  The argument against this decision would be the first inclusion of the logger in the ulits.cc file. 
```suggestion
                MLPD_INFO("%c: pushing %zu back to vector %zu\n", sym, g, f);
```

Reply:
This is a debug leftover. I will remove it completely since it is not an essential print.